README.md

```diff
@@ -169,11 +169,21 @@ $ git clone https://github.com/vhda/verilog_systemverilog.vim
 ### Indent options
 
 * __`b:verilog_indent_width`__ - Use this variable to override the option `&shiftwidth`.
-* __`b:verilog_indent_modules`__ - Indent code after module declaration.
 * __`b:verilog_indent_preproc`__ - Indent code after preprocessor statements.
 * __`b:verilog_dont_deindent_eos`__ - Keep last `)` of module port declaration indented.
 * __`b:verilog_indent_assign_fix`__ - Indent assignments by fixed amount.
 
```

Comment:
Please remove this empty line. Something is broken in the HTML output in GitHub and I don't know why.